lib/hexpm_web/endpoint.ex

```diff
@@ -4,10 +4,12 @@ defmodule HexpmWeb.Endpoint do
   plug HexpmWeb.Plugs.Forwarded
 
   @session_options [
```

Comment:
I think we should make use of `same_site: "Strict"` here, maybe a good reason not to?

Reply:
👍